mahta/JS/JS DOM-manipulation/script.js

```diff
@@ -0,0 +1,116 @@
+function generateRandomColor(y, b) {
+  const yellow = Math.floor(Math.random() * (y - 155) + 155);
+  const blue = Math.floor(Math.random() * (b - 100) + 100);
+  return `rgb(${yellow}, ${yellow}, ${blue})`;
+}
```

Comment:
It's okay. Additionally, you can use the `HSL` Color Code to generate a random color.